specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/Prediction.json

```diff
@@ -0,0 +1,917 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
+    "title": "Custom Vision Prediction Client"
+  },
+  "host": "southcentralus.api.cognitive.microsoft.com",
+  "basePath": "/customvision/v3.0/prediction",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/{projectId}/classify/models/{publishedModelName}/url": {
+      "post": {
+        "tags": [
+          "ImagePredictionApi"
+        ],
+        "summary": "Classify an image url and saves the result.",
+        "operationId": "ClassifyImageUrl",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "name": "projectId",
```

Comment:
In general, using uuids/guids to identify resources tend to make APIs/endpoints harder to use. They are great for machine-readable only things, but assuming that a project is something that a user would create explicitly, giving it a name would allow them to convey some context to which project it was.

Reply:
The project object contains a name and is displayed in our portal. The ids are used in the paths and not something we can change at this time.